package.json

```diff
@@ -50,6 +50,7 @@
     "html-minifier": "^3.0.2",
     "jsonapi-serializer": "^3.5.3",
     "jsonfile": "^3.0.0",
+    "mocha": "^4.0.1",
```

Comment:
Why do we have installed mocha here ? @agbilotia1998

Reply:
I was testing if global installation of mocha is causing problem in travis build or not.